### ISSUE TITLE
Unnecessary D102 for property setter if getter has already a docstring

### ISSUE THREAD

**Issue by oefe** [charliermarsh/ruff#1873]
When a property has a setter and a getter, it should be enough to write a docstring for the getter only. After all, the property has only a single docstring. However, Ruff issues a D102 warning about the missing docstring for the setter.

Pydocstyle doesn't complain about this.

### Example 

file `property_with_setter.py`:
```python
class PropertyWithSetter:
    @property
    def foo(self) -> str:
        """Docstring for foo."""
        return "foo"

    @foo.setter
    def set_foo(self, value: str) -> None:
        pass
```

```
>ruff property_with_setter.py --isolated --select D102
property_with_setter.py:8:9: D102 Missing docstring in public method
Found 1 error(s).

>ruff --version
ruff 0.0.221
```

**Comment by charliermarsh** [comment]
Sounds like an oversight! Thanks for filing.
